src/utils.rs

```diff
@@ -0,0 +1,163 @@
+use crate::{Result, Size};
+use std::cmp;
+
+#[cfg(feature = "image")]
+use fast_image_resize as fr;
+#[cfg(feature = "image")]
```

Comment:
IMO ou can remove all cfg feature image here and put it on mod utils;

Reply:
ok thanks ! yes I wanted to asking you if I could put the cfg feature on an entire mod 🙂